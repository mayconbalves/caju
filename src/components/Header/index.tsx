import { Wrapper } from './styles'
import { HeaderProps } from './types'

const Header = ({ children }: HeaderProps) => {
  return <Wrapper>{children}</Wrapper>
}

export default Header
