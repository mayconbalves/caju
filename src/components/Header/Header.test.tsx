import { render, screen } from '@testing-library/react'
import Header from '.'

describe('Header component', () => {
  it('should render children element correctly', () => {
    const { container } = render(
      <Header>
        <h1>Caju Front Teste</h1>
      </Header>
    )

    const headerElement = screen.getByText('Caju Front Teste')
    expect(headerElement).toBeInTheDocument()

    expect(container).toMatchSnapshot()
  })
})
