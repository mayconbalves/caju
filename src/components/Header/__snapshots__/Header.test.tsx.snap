// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render children element correctly 1`] = `
<div>
  <header
    class="sc-aYaIB jFxYKq"
  >
    <h1>
      Caju Front Teste
    </h1>
  </header>
</div>
`;
